### ISSUE TITLE
Privide a count of images waiting to be transcribed

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#1727]

**Comment by edickens** [comment]
The impression is that we are short of work, but there are plenty of images, if not always where needed.  I'd like to know the true figure so that we can use this to get transcribers to ask to do them, and also to recruit more transcribers.
A count of the number of images marked as "Unallocated" to be displayed in the home page footer.  This will show what we have and also show researchers that we have a long way to go and could do with help with transcribing.
Would it also be worth including images that are "Allocated"?  This means that they are under the control of the Syndicate Coordinator, but have not yet been "Assigned" to a transcriber.
I think  we may have as many as 100,000 images waiting to be transcribed.
This count will also prompt County Coordinators to get their images properly classified if they have in fact been transcribed.

**Comment by Captainkirkdawson** [comment]
Very interesting results. There are 836934 images on the image server. Of those 243676 are unallocated. There are 408795 for which the status has not been set! Even if we assumed that all of the latter are completed (unreasonable) then over 30% of what we have on hand is available for transcription.

**Comment by edickens** [comment]
I think one problem is that when we first set up the image server, I set the first few counties as Unallocated until we decided to leave the as having no status.  So some of the Unallocated will have been done.  It will be interesting to see how it goes by County.
Hopefully this will encourage CCs to get the status set correctly so that we can then recruit transcribers to do what we know has not been done.
Thanks Kirk
